src/docs/markdown/caddyfile/concepts.md

```diff
@@ -178,22 +178,20 @@ If you specify a hostname, only requests with a matching Host header will be hon
 
 Wildcards (`*`) may be used, but only to represent precisely one label of the hostname. For example, `*.example.com` matches `foo.example.com` but not `foo.bar.example.com`, and `*` matches `localhost` but not `example.com`. To catch all hosts, omit the host portion of the address.
 
-If multiple sites share the same definition, you can list all of them together:
+If multiple sites share the same definition, you can list all of them together, separated by spaces or commas:
```

Comment:
This is not entirely clear, since you always need spaces, you cannot separate them with commas.